src/base.jl

```diff
@@ -46,8 +46,22 @@ function ShortString{T}(s::ShortString{S}) where {T, S}
     ShortString{T}(content | T(sz))
 end
 
+const shft_int = UInt === UInt32 ? 2 : 3
 
-String(s::ShortString) = String(reinterpret(UInt8, [s.size_content|>ntoh])[1:sizeof(s)])
+function String(s::ShortString{T}) where {T}
+    len = sizeof(s)
+    len === 0 && return ""
+    val = ntoh(s.size_content)
+    sv = Base.StringVector(len)
+    # Loop over UInt64 sized chunks here
```

Comment:
```suggestion
    # Loop over UInt sized chunks here
```

Reply:
can we add a benchmark for both to prevent regression and point to that benchmark. So we can see how much faster this is?